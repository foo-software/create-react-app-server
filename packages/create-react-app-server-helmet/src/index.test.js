import * as packageExports from '.';

describe('@foo-software/create-react-app-server-helmet', () => {
  it('should match snapshot', () => {
    expect(packageExports).toMatchSnapshot();
  });
});
