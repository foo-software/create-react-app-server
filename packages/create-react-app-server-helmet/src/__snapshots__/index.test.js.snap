// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@foo-software/create-react-app-server-helmet should match snapshot 1`] = `
Object {
  "CreateReactAppServerHelmet": [Function],
  "CreateReactAppServerHelmetContext": Object {
    "$$typeof": Symbol(react.context),
    "Consumer": Object {
      "$$typeof": Symbol(react.context),
      "_calculateChangedBits": null,
      "_context": [Circular],
    },
    "Provider": Object {
      "$$typeof": Symbol(react.provider),
      "_context": [Circular],
    },
    "_calculateChangedBits": null,
    "_currentRenderer": null,
    "_currentRenderer2": null,
    "_currentValue": "create-react-app-server-helmet-context",
    "_currentValue2": "create-react-app-server-helmet-context",
    "_threadCount": 0,
  },
  "withCreateReactAppServerHelmetProvider": [Function],
}
`;
